erdpy/projects/eei_checks.py

```diff
@@ -0,0 +1,65 @@
+import logging
+from typing import List
+from erdpy import utils
+from erdpy.errors import NotSupportedProjectFeature
+from erdpy.projects.eei_activation import ActivationKnowledge
+from erdpy.projects.eei_registry import EEIRegistry
+from erdpy.projects.interfaces import IProject
+
+logger = logging.getLogger("eei")
+
+MAINNET_PROXY_URL = "https://gateway.elrond.com"
+MAINNET_ENABLE_EPOCHS_URL = "https://raw.githubusercontent.com/ElrondNetwork/elrond-config-mainnet/master/enableEpochs.toml"
+DEVNET_PROXY_URL = "https://devnet-gateway.elrond.com"
+DEVNET_ENABLE_EPOCHS_URL = "https://raw.githubusercontent.com/ElrondNetwork/elrond-config-devnet/master/enableEpochs.toml"
+
+
+def check_compatibility(project: IProject):
+    if _should_skip_checks(project):
+        return
+
+    logger.info("check_compatibility")
+
+    wasm_file = project.get_file_wasm()
+    imports_file = wasm_file.with_suffix(".imports.json")
+    imports = utils.read_json_file(imports_file)
+
+    compatible_with_mainnet = _check_imports_compatibility(imports, ActivationKnowledge("mainnet", MAINNET_PROXY_URL, MAINNET_ENABLE_EPOCHS_URL))
+    compatible_with_devnet = _check_imports_compatibility(imports, ActivationKnowledge("devnet", DEVNET_PROXY_URL, DEVNET_ENABLE_EPOCHS_URL))
```

Comment:
Consider moving the `ActivationKnowledge` constructors on separate lines with assignments to variables.

Reply:
Fixed.